controllers/servers/host_definer/settings.py

```diff
@@ -44,3 +43,10 @@
 DYNAMIC_NODE_LABELING_ENV_VAR = 'DYNAMIC_NODE_LABELING'
 ALLOW_DELETE_ENV_VAR = 'ALLOW_DELETE'
 SECRET_SEPARATOR = ','
+DEFINE_ACTION = 'Define'
+UNDEFINE_ACTION = 'Undefine'
+SUCCESS_MESSAGE = 'Host defined successfully on the array'
+FAILED = 'Failed'
+SUCCESSFUL = 'Successful'
+NORMAL = 'Normal'
```

Comment:
can we? (and in other relevant places)
```suggestion
NORMAL_EVENT_TYPE = 'Normal'
```

Reply:
and also change in [warnning event type](https://github.com/IBM/ibm-block-csi-driver/blob/3081b578be394557f06a040651c1f24389de80a9/controllers/servers/host_definer/settings.py#L52)?

Reply:
generally, for all of [those](https://github.com/IBM/ibm-block-csi-driver/blob/3081b578be394557f06a040651c1f24389de80a9/controllers/servers/host_definer/settings.py#L49-L52), add any description so no one will ask himself what they are for..

Reply:
done